combat-trainer.lic

```diff
@@ -964,7 +964,7 @@ class TrainerProcess
     @no_app = []
 
     @dont_stalk = settings.dont_stalk
-
+	@stun_weapon = settings.stun_weapon
```

Comment:
This needs an echo after it (similar to line 951). While you're in here, could you add two more echos after lines 968 and 969?